tests/transformers/test_transformers_autolog.py

```diff
@@ -352,6 +364,10 @@ def test_transformers_autolog_adheres_to_global_behavior_using_trainer(transform
     assert len(runs) == 1
 
 
+@pytest.mark.skipif(
+    Version(transformers.__version__).is_devrelease,
+    reason="fails with error: 'CallbackHandler' object has no attribute 'tokenizer'"
```

Comment:
can we fix this error?

Reply:
I think we need to update setfit code for fixing it

Reply:
Sure.

Reply:
https://github.com/huggingface/setfit/issues/564

Reply:
@WeichenXu123 you can include this in `reason`.